app/scripts/components/form/search.js

```diff
@@ -48,20 +64,27 @@ class Search extends React.Component {
 
   render () {
     return (
-      <div className='filter__item'>
-        <form className='search__wrapper form-group__element' onSubmit={this.submit} value={this.state.value}>
-          <input className='search' type='search' onChange={this.complete}/>
-          <span className='search__icon'/>
+        <div className='filter__item'>
```

Comment:
Why remove the indentation on this line and line 72?

Reply:
emacs. autoformating problems I would guess.

Reply:
e382c25